Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
```

Comment:
Why `simple` here?

Reply:
It's default config for pipenv. And other source url are usually the same.